app/controllers/api/v1/works_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Api::V1::WorksController < Api::V1::BaseController
+  respond_to :json
+
+  # Return the URLs of a batch of individual works. Limits the number of URLs to
+  # IMPORT_MAX_CHAPTERS so it doesn't get tied up in checking URLs for too long.
+  # Params:
+  # +original_urls+:: an array of original URLs to find on the Archive
+  def batch_urls
```

Comment:
Well, it lowered the score but didn't get rid of the message! I think I'm going to leave it at that...